galeracluster/source/documentation/schema-upgrades.rst

```diff
@@ -61,12 +62,14 @@ Schema Upgrades
 
 Schema changes are of particular interest related to Galara Cluster. Schema changes are  :abbr:`DDL (Data Definition Language)` statement executed on a database (e.g., ``CREATE TABLE``, ``GRANT``).  These :abbr:`DDL (Data Definition Language)` statements change the database itself and are non-transactional.
 
-Galera Cluster processes schema changes by two different methods:
+Galera Cluster processes schema changes by three different methods:
 
 - :ref:`Total Order Isolation <toi>`: Abbreviated as TOI, these are schema changes made on all cluster nodes in the same total order sequence, preventing other transations from committing for the duration of the operation.
 
 - :ref:`Rolling Schema Upgrade <rsu>` Known also as RSU, these are schema changes run locally, affecting only the node on which they are run.  The changes do not replicate to the rest of the cluster.
 
+- :ref:`Non-Blocking Operations <nbo>`: Abbreviated as NBO, these are schema changes made on all cluster nodes in the same total order sequence, preventing other transations from committing for the duration of the operation, with mush more efficient locking strategy that the TOI method.
+
 You can set the method for online schema changes by using the ``wsrep_OSU_method`` parameter in the configuration file, (``my.ini`` or ``my.cnf`, depending on your build) or through the ``mysql`` client.  Galera Cluster defaults to the Total Order Isolation method.
 
 .. note:: If you're using Galera Cluster for Percona XtraDB Cluster, see the the `pt-online-schema-change <https://www.percona.com/doc/percona-toolkit/2.2/pt-online-schema-change.html>`_ in the Percona Toolkit.
```

Comment:
I think it is worth updating it to the latest release https://www.percona.com/doc/percona-toolkit/3.0/pt-online-schema-change.html